samples/compiler_plugins/xnnpack/src/xnnpack/Conversion/StablehloToXnnpack.cpp

```diff
@@ -183,8 +183,19 @@ class ConvertFullyConnectedLayer
       DenseIntElementsAttr permAttr = rewriter.getI64TensorAttr(perm);
       info.kernel = rewriter.create<stablehlo::TransposeOp>(
           op.getLoc(), info.kernel, permAttr);
+      kernelType = info.kernel.getType().cast<RankedTensorType>();
     }
 
+    auto offsetType = RankedTensorType::get(kernelType.getShape(),
+                                            rewriter.getIntegerType(8));
+    int8_t offsetInt = 8;
+    Value offset = rewriter.create<stablehlo::ConstantOp>(
+        op.getLoc(), DenseIntElementsAttr::get(offsetType, offsetInt));
```

Comment:
Can we create the offset directly with the kernel type?

Reply:
I did try that. The issue is that the `offsetInt` value needs to have the same element type as the `offsetType`, and I don't know how to create an `int4` in C++.

The compiler should easily optimize this away, though.

Reply:
Good to know. Thanks for the info.